cluster/src/test/java/org/apache/iotdb/cluster/log/manage/RaftLogManagerTest.java

```diff
@@ -48,12 +52,22 @@ public void apply(Log log) {
 			appliedLogs.add(log);
 		}
 	};
+	private int testIdentifier = 1;
 
 	@Before
 	public void setUp() {
 		appliedLogs = new HashSet<>();
 	}
 
+	@After
+	public void tearDown() {
+		File dir = new File(new SyncLogDequeSerializer(testIdentifier).getLogDir());
+		for (File file : dir.listFiles()) {
+			file.delete();
+		}
+		dir.delete();
+	}
```

Comment:
Please check the indents.

Reply:
It seems wired.In my local editor,the indents are properly.Maybe it ’s a webpage display problem.